extensions/amp-list/0.1/amp-list.js

```diff
@@ -48,10 +54,35 @@ export class AmpList extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    return this.populateList_();
+  }
+
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const srcMutation = mutations['src'];
+    if (srcMutation != undefined) {
+      const p = this.populateList_();
+      if (getMode().test) {
+        this.mutationPromise_ = p;
```

Comment:
Can we instead try to get what we need by stubbing `this.populateList_` in the test instead`?